main.tf

```diff
@@ -242,7 +242,7 @@ resource "google_container_node_pool" "node_pools" {
     }
 
     tags = [
-      "iap-tcp-forwarding-allowed",
+      "iap_tcp_forwarding-allowed",
```

Comment:
I guess this was underscored to match the resource name, but since this is an admin-visible tag do we really want that to be the case? I feel like its formatting should be geared toward the end-user rather than the matching the terraform code.

Reply:
I guess this is the same topic as one of steve's comments, but just a different tag?